lib/mocha.js

```diff
@@ -110,11 +121,13 @@ function Mocha(options) {
 }
 
 /**
- * Enable or disable bailing on the first failure.
+ * Enables or disables bailing on the first failure.
  *
  * @public
- * @api public
- * @param {boolean} [bail]
+ * @see {@link https://mochajs.org/#-b---bail|CLI option}
+ * @param {boolean} [bail=true] - Whether to bail on first error.
```

Comment:
Is `bail` really true by default? This doesn't match the default CLI behavior

Reply:
It doesn't?
```js
Mocha.prototype.bail = function(bail) {
  if (!arguments.length) {
    bail = true;
  }
  this.suite.bail(bail);
  return this;
};
```

Reply:
Bear in mind, I was documenting _intent_, not buggy behavior. And yes, there are plenty I found while documenting just this file.